admin/src/main/resources/io/buoyant/admin/js/src/combined_client_graph.js

```diff
@@ -89,6 +103,14 @@ define([
           ignoredClients[routerName][client] = false;
         },
 
+        expireClient: function(client) {
+          expiredClients[routerName][client] = true;
```

Comment:
default is clients expire after being idle for 10 minutes.  but it's configurable.